api/integrations_aws.go

```diff
@@ -146,6 +146,8 @@ type AwsIntegrationData struct {
 
 	// AwsAccountID is the AWS account that owns the IAM role credentials
 	AwsAccountID string `json:"AWS_ACCOUNT_ID,omitempty" mapstructure:"AWS_ACCOUNT_ID"`
+
+	GovCloudCredentials AwsGovCloudCreds `json:"ACCESS_KEY_CREDENTIALS,omitempty" mapstructure:"ACCESS_KEY_CREDENTIALS"`
```

Comment:
You will have to add `omitempty` to the `CROSS_ACCOUNT_CREDENTIALS ` field. If not,
we will have a Schema Validation failure. (I think)